### PR TITLE
Feature/16 create map class

### PR DESCRIPTION
**Pull request by Erez-Shlingbaum**

### CODE REVIEW

File: src/empire/service_locator.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+As more services will be added, this module will be responsible for locating them
+Separating the serviced class from knowing how or where a service is implemented
+"""
+
+import config
+
+
+def get_game_config() -> config.GameConfig:
+    return config.GameConfig('Empire', 1500, 900, 60)
```
Comment (segalmatan) [inline src/empire/service_locator.py]: put it in constants, so I wont forget which is height and which is width :)

File: src/empire/main.py [hunk 2]
```diff
@@ -21,15 +23,16 @@ def setup_logging(log_level: int):
     )
 
 
-def main(args):
+def main(args: List[str]):
```
Comment (segalmatan) [inline src/empire/main.py]: remove it, were not using it

Comment (segalmatan) [reply src/empire/main.py]: I mean args

File: src/empire/game.py [hunk 3]
```diff
@@ -1,70 +1,42 @@
-import dataclasses
+import logging
 
 import pygame
 
+import service_locator
```
Comment (segalmatan) [inline src/empire/game.py:5]: is it used?

File: src/empire/__init__.py [hunk 4]
```diff
@@ -1,18 +0,0 @@
-import sys
-
-if sys.version_info[:2] >= (3, 8):
-    # TODO: Import directly (no need for conditional) when `python_requires = >= 3.8`
-    from importlib.metadata import PackageNotFoundError, version  # pragma: no cover
-else:
-    from importlib_metadata import PackageNotFoundError, version  # pragma: no cover
-
-try:
```
Comment (segalmatan) [inline src/empire/__init__.py:9]: any idea on what all of this does?

File: src/empire/game.py (hunk 3 above)
Comment (Erez-Shlingbaum) [reply src/empire/game.py:5]: yes
        self.config = service_locator.get_game_config()

File: src/empire/__init__.py (hunk 4 above)
Comment (Erez-Shlingbaum) [reply src/empire/__init__.py:9]: Defines dist_name and version of our package. We don't really need this anyway

**Review by segalmatan — APPROVED** [review]

**Comment by Erez-Shlingbaum** [comment]
Closes #16

**Pull request merged**